pkg/commands/server/plugins.go

```diff
@@ -62,3 +63,34 @@ func cmdPlugins(c *cli.Context) error {
 	}
 	return formatter.Write()
 }
+
+// getPlugins is a helper function that takes the given plugin tag and returns
+// the set of matched plugins.
+// FIXME: getPlugins is actually not consumed by cmdPlugins in this file
+// because `plugins` works without arguments. It is used by `plugins
+// info` and `plugins health`, which are in other files. Not sure if we
+// should keep it here or not. In the cmdPlugins, instead of calling
+// client.Client.Plugin(), we can provide an empty string as the first
+// parameter for getPlugins, like so getPlugins("", c), to have the same effect
+// and make use of getPlugins for consistency among all the plugins command.
+// Yet it doesn't look as pretty.
```

Comment:
Good note!

As for where this should exist, I think that it is fine living here. It could also move to some utility file, but I don't think thats necessary. 

Another option for how this could be handled in a generic/consistent way is to pass in plugin tags as a variadic argument to the function (see also: https://gobyexample.com/variadic-functions). Basically, you could do something like:
```go
func getPlugins(c *cli.Context, tags ...string) ([]scheme.Plugin, error) {
    var plugins []scheme.Plugin
    
    ...
}
``` 

This means that you could call the function like:
```go
// no tags
getPlugins(c)

// one tag
getPlugins(c, "vaporio/foo")

// multiple tags
getPlugins(c, "vaporio/foo", "vaporio/bar")
```

This could be used in all cases because it would get the plugins, but only filter them if tags are provided, so this could be a good route to try.